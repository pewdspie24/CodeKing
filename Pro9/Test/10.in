10
2790728624 3756896738
4094380565 8192145460
52586776 1710331809
4294780786 4481742833
2289162362 2413662313
962422971 1518831956
3341173248 6263290970
3121926903 5211281775
3885933608 5885674147
467783445 1476533356
