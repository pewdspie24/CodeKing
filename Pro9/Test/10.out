10112278
29516732
19276543
2247717
1623768
7454103
24394547
19799537
15831391
11229361
